### PR TITLE
Ch14271/fix leadconduit integrations ci test error

### PR DESCRIPTION
**Pull request by jarrmill**
This repo hasn't been passing its CI/CD tests due to a Codecov error. Codecov runs the following command, `npx nyc mocha test; npx nyc report --reporter=html; open ./coverage/index.html`, and throws an error when it hits the `open ./coverage/index.html` bit. See [this Clubhouse ticket]() for the error log information.

This PR adds another package.json script, `test:coverage:lcov,` which is what our other repos use in their ci/cd scripts (see [ leadconduit-mappings' test yml](https://github.com/activeprospect/leadconduit-mapping/blob/a069892cc8cd089e12017c332ab04b5a9ecc7d7d/.github/workflows/test.yml#L25) for reference). The first command, `test:coverage` is meant to be run by programmers, not github actions, and I think this is what caused the error.

Thanks in advance.

### CODE REVIEW

**Review by chasingSublimity — APPROVED** [review]
Whoops, sorry about that. LGTM! 🚀

**Comment by jarrmill** [comment]
@chasingSublimity I think this would have been caught earlier if SalesRabbit hadn't been erroring out. So it goes. TY!

File: package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
   "main": "index.js",
   "scripts": {
     "test": "mocha",
-    "test:coverage": "npx nyc mocha test; npx nyc report --reporter=html; open ./coverage/index.html"
+    "test:coverage": "npx nyc mocha test; npx nyc report --reporter=html; open ./coverage/index.html",
```
Comment (cgrayson) [inline package.json:9]: Ah, I bet that `open` was the problem. Might be handy to open the HTML report locally, less so for a background CI job at Github. 😄

**Review by cgrayson — APPROVED** [review]
LGTM

**Pull request merged**